frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/state.ts

```diff
@@ -8,6 +8,7 @@ export const initialState = {
   diskType: 'All',
   diskMode: diskModeDropdownItems.BLOCK,
   deviceType: [],
+  fsType: '',
```

Comment:
Shouldn't we default to something?

Reply:
> Shouldn't we default to something?
The default value can be anything based on the configuration. So we are not showing any default. If nothing selected, the back-end will decide the correct or applicable default fstype based on the operating system and other parameters like other existing LVS fstype and so on.